README.md

```diff
@@ -38,3 +47,22 @@ Handwrite can generate custom font of your handwriting based on a writing sample
 `python setup.py test`
 
 You are ready to go!
+
+# Usage
+
+## Creating your Handwritten Sample 
+
+1. Take a printout of `handwrite_sample.pdf`, present in the cloned `handwrite` folder.
```

Comment:
Since this will be package, not everybody will clone the project. Some might pip install. 

Better would be to just paste the link to file.
```suggestion
1. Take a printout of the [sample form](https://github.com/cod-ed/handwrite/raw/main/handwrite_sample.pdf).
```